docs/faq.rst

```diff
@@ -330,6 +330,33 @@ needs to be ascertained and fixed.
 issues. We recommend to first run without ``--repair`` to assess the situation.
 If the found issues and proposed repairs seem right, re-run "check" with ``--repair`` enabled.
 
+How probable is it to get a hash collision problem?
+---------------------------------------------------
+
+If you noticed, there are some issues (:issue:`170` (**warning: hell**) and :issue:`4884`)
+about the probability of a chunk having the same hash as another chunk, making the file
+corrupted because it grabbed the wrong chunk. This is called the `Birthday Problem
+<https://en.wikipedia.org/wiki/Birthday_problem>`_.
+
+There is a lot of probability in here so, I can give you my interpretation of
+such math but it's honestly better that you read it yourself and grab your own
+resolution from that.
+
+Assuming that all your chunks have a size of :math:`2^{21}` bytes (approximately 2.1 MB)
+and we have a "perfect" hash algorithm, we can think that the probability of collision 
+would be of :math:`p^2/2^{n+1}` then, using SHA-256 (:math:`n=256`) and for example
+we have a billion chunks (:math:`p=10^9`) (a billion chunks would be about 2.1ZB).
+The probability would be around to 4.3e-60.
+
+Still, the real question is if Borg tries to not make this happen?
+
+Well... it used to not check anything but there was a feature added which saves the size
+of the chunks too, so the size of the chunks is compared to the size that you got with the
+hash, this doesn't save us from everything but reduces the chances of corrupting the file.
```

Comment:
hmm, iirc it just raises an exception. so it will just not silently corrupt a file, but still not result in a correct file, right?